packages/web-components/fast-foundation/src/radio-group/radio-group.ts

```diff
@@ -148,17 +167,13 @@ export class RadioGroup extends FASTElement {
             radioButtons[0].setAttribute("tabindex", "0");
             this.focusedRadio = radioButtons[0];
         }
-
-        this.parentToolbar = this.parentElement?.closest('[role="toolbar"]');
-        this.isInsideToolbar =
-            this.parentToolbar !== undefined && this.parentToolbar !== null;
     }
 
     private getFilteredRadioButtons = (): RadioControl[] => {
         const radioButtons: RadioControl[] = [];
         if (this.slottedRadioButtons !== undefined) {
             this.slottedRadioButtons.forEach((item: any) => {
-                if (item instanceof HTMLElement) {
+                if (isRadioElement(item)) {
```

Comment:
If you wanted, you can use this directly for the filter and the slottedRadioButtons would then this test wouldn't be needed here. Not sure if that works for your scenario or not but wanted to mention it.

Reply:
Ok, I'll put that in and see if I can't clean it up a bit after